### PR TITLE
fix: improve readme

### PR DESCRIPTION
**Pull request by axel7083**
Fixes https://github.com/redhat-developer/podman-desktop-redhat-pack-ext/issues/6

### CODE REVIEW

**Review by benoitf — COMMENTED** [review]
should we avoid the usage of 'you' ?

**Review by benoitf — APPROVED** [review]

**Comment by axel7083** [comment]
> should we avoid the usage of 'you' ?

I have no idea, I am not sure of the guidelines for the readme.. @slemeur

**Comment by benoitf** [comment]
Need to add documentation for the OpenShift image checker

https://github.com/redhat-developer/podman-desktop-redhat-pack-ext/pull/10

File: README.md [hunk 1]
```diff
@@ -1,20 +1,39 @@
 # Red Hat Extension Pack for Podman Desktop
 
-The Podman Desktop Extension Pack includes various Red Hat extensions designed to enhance Podman Desktop with Red Hat technologies and tools.
+The Podman Desktop Extension Pack is a collection of extensions designed to enhance the functionality of Podman Desktop with Red Hat technologies and tools. With this pack, you can access a range of features that will help you streamline your development workflow and improve your productivity.
```
Comment (slemeur) [inline README.md:3]:
```suggestion
The Podman Desktop Extension Pack is a collection of extensions designed to enhance the functionality of Podman Desktop with Red Hat technologies and tools. This pack gives access to a range of features streamlining development workflow, improving productivity and brings Red Hat tools handy.
```

**Review by slemeur — APPROVED** [review]

**Pull request merged**